src/flb_sds.c

```diff
@@ -33,7 +33,7 @@
 #include <stdarg.h>
 #include <ctype.h>
 
-static flb_sds_t sds_alloc(size_t size)
```

Comment:
This collides with the definition in cfl/cfl.h. This caused problems once I added the cmetrics import to flb_log.c.